Uni_life_tracker/app/src/main/java/com/generic/ult/SettingsActivity.java

```diff
@@ -1,19 +1,40 @@
 package com.generic.ult;
 
+import Database.UserInfoDatabaseHelper;
+import android.os.Bundle;
 import android.view.View;
 import androidx.appcompat.app.AppCompatActivity;
-import android.os.Bundle;
 import androidx.drawerlayout.widget.DrawerLayout;
+import com.google.android.material.textfield.TextInputEditText;
+import com.google.android.material.textfield.TextInputLayout;
+
+import java.util.Objects;
 
 public class SettingsActivity extends AppCompatActivity {
     // initialize variable
     DrawerLayout drawerLayout;
 
+    private TextInputEditText etName;
+    private TextInputEditText etUni;
+    private TextInputLayout tiName;
+    private TextInputLayout tiUni;
+
```

Comment:
a really minor detail, but if you want you can declare these all in 1 line to save a few lines